mailserver/limiter_test.go

```diff
@@ -0,0 +1,100 @@
+package mailserver
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+)
+
+func TestLimiterSuite(t *testing.T) {
+	suite.Run(t, new(LimiterSuite))
+}
+
+type LimiterSuite struct {
+	suite.Suite
```

Comment:
not sure if it makes sense to use suites if there is no setup/teardown
the reason for not using them is that sub-tests are filterable

Reply:
Yep, you're right. Limiter tests are simple enough to run them without a suite, thats fixed now :)